util/src/main/java/com/netcracker/ConsoleHelper.java

```diff
@@ -0,0 +1,39 @@
+package com.netcracker;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class ConsoleHelper {
+    private static final BufferedReader READER = new BufferedReader(new InputStreamReader(System.in));
+
+    private ConsoleHelper(){
+        throw new RuntimeException("Don't create instance of this class");
+    }
+
+    public static void writeMessage(String message){
```

Comment:
Hm, that is weird ... the bracket { at the end of the line should have additional space before it:
){ -> ) {
How long ago did you update IDEA (might be that default setting changed) ? Or may be you customized code formatting by yourself ? Cause I use IDEA downloaded like two weeks ago and it formats code slightly differently :)